patches/kaggle_gcp.py

```diff
@@ -57,10 +57,9 @@ def refresh(self, request):
             raise RefreshError('Unable to refresh access token due to connection error.') from e
         except Exception as e:
             Log.error(f"Error trying to refresh access token: {e}")
-            if (not get_integrations().has_bigquery() and self.target == GcpTarget.BIGQUERY):
-                Log.error(f"No bigquery integration found.")
-                print(
-                    'Please ensure you have selected a BigQuery account in the Kernels Settings sidebar.')
+            Log.error(f"No {self.target.service} integration found.")
```

Comment:
I think we still need to have the check we had before but add it for GCS as well, because this message is only for the case that the integration isn't found.
ie.
if ((not get_integrations().has_bigquery() and self.target == GcpTarget.BIGQUERY) or (not get_integrations().has_gcs() and self.target == GcpTarget.GCS)):

or we could add a mapping of GcpTarget enum<->string value.